docs/blog/v5-intro.md

```diff
@@ -0,0 +1,167 @@
+# Version 5 RC
+
+If you are reading this it means that we have finally got our first RC of version 5 out of the gates.
+The maintainers is extremely happy to finally being able to do so.
+
+In this blog we will try to explain the changes that we have made and the future of the operator.
+In [v5 getting started](v5-getting-started.md), you will be able to read how to try out the operator.
+
+Version 5 or v5 as I will write from now on is a complete rewrite of version 4 of the operator.
+The API versions are different so if you want you can run both the operators at the same time while you do a migration.
+
+There are multiple reasons why we did this, but the main reason is the number of architectural changes that we have done.
+
+Just a reminder this is an RC and not production ready, we will probably do a few breaking changes before releasing v5 but we really want to bring you something to try out and get some feedback on what we have done.
+
+## Architectural changes
+
+The main change is that instead of grafana "finding" the dashboards, data sources CR they now find the grafana instance.
```

Comment:
```suggestion
The main change is that now, instead of grafana "finding" the dashboard and datasource CRs, they will now find the grafana instance.
```

Reply:
I got similar feedback from a friend, I rewrote it a bit. Can you take a second look and see if you think it looks better?

Reply:
Yep, new version looks better, reading the original wording seemed a bit awkward, but I couldn't put my finger on what it was exactly, Thanks! :+1: :sweat_smile: